src/ft_strtok.c

```diff
@@ -0,0 +1,155 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_strtok.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: minakim <minakim@student.42berlin.de>      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/07/18 19:59:27 by minakim           #+#    #+#             */
+/*   Updated: 2023/07/18 21:48:04 by minakim          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../libft/include/libft.h"
+//#include "libft.h"
+
+/**
+ * @note These functions have been 'briefly' tested.
+ * The three functions below, "ft_strpbrk, ft_strspn, and ft_strtok" have
+ * connectivity and can be used together.
+ */
+char	*ft_strpbrk(const char *str, const char *delim);
+size_t	ft_strspn(const char *str, const char *delim);
+char	*ft_strtok(char *str, const char *delim);
+
+
+/**
```

Comment:
I am not clear but, it seems you implemented `strcspn`, not `strspn`. Let's take a look together tomorrow.